Bottom: 0131d96d177020040fd084b7ad5e8d73b6b6b62f
Top:    c69be97cb7c69bbbb56899d08d1862a857e0a249
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-22 12:18:36 +0000

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 230b8e5..c452997 100644
--- a/configure.ac
+++ b/configure.ac
@@ -392,8 +392,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
